gradle-baseline-java/src/main/groovy/com/palantir/baseline/extensions/BaselineErrorProneExtension.java

```diff
@@ -38,6 +38,8 @@
             "ExtendsErrorOrThrowable",
             "FinalClass",
             "ImplicitPublicBuilderConstructor",
+            "JavaTimeDefaultTimeZone",
+            "JavaTimeSystemDefaultTimeZone",
```

Comment:
Nit: can we move this one from ERROR->WARNING since there are cases (non-server) where the users timezone is preferred? If we're auto-fixing it, there's not a great deal of difference between the two.

Reply:
I changed this in response to https://github.com/palantir/gradle-baseline/pull/1531#discussion_r517019478